pydicom/tests/test_uid.py

```diff
@@ -1,93 +1,22 @@
-# test_uid.py
 """Test suite for uid.py"""
-# Copyright (c) 2008-2012 Darcy Mason
-# This file is part of pydicom, released under a modified MIT license.
-#    See the file LICENSE included with this distribution, also
-#    available at https://github.com/pydicom/pydicom
-
```

Comment:
From what I've read elsewhere the convention is that copyright notices (and the # filename.py line) shouldn't be placed within the files. I've been removing them when I can remember to.

I just think it looks cleaner without them and the copyright is in the LICENSE file.